src/utils/file.ts

```diff
@@ -19,34 +19,6 @@ export function getFileNameWithoutExtension(fileName: string) {
   return fileName.replace(/\.[^/.]+$/, '')
 }
 
-export function isRecording(fileName: string) {
-  return fileName.endsWith('.har')
-}
-
-export function isGenerator(fileName: string) {
```

Comment:
With the addition of test data, this is no longer accurate. Instead, we get `StudioFile` objects from the main process, so we can always check the type directly